packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -1144,12 +1236,40 @@ class SliverMultiBoxAdaptorElement extends RenderObjectElement implements Render
     _didUnderflow = value;
   }
 
+  /// check if the given child is actually in rendering tree.
```

Comment:
I think even the keptAlive children are still included in the render tree, no? If you print the render tree for an app, the keepAlive children show up in there.

Reply:
(Same confusion on my side as in the other comment, feel free to ignore (or clarify the comment)).

Reply:
unpdated doc comment